gremlin/agent_apparmor.profile

```diff
@@ -0,0 +1,100 @@
+#include <tunables/global>
+
+
+profile gremlin-agent flags=(attach_disconnected,mediate_deleted) {
+  #include <abstractions/base>
+
+  network inet tcp,
+  network inet udp,
+  network inet icmp,
+
+  deny network raw,
+
+  deny network packet,
+
+  file,
+  umount,
+
+  /entrypoint.sh rx,
+  /var/log/gremlin-api/** rwx,
```

Comment:
Is this actually a directory we write logs to? I can't seem to find it in the agent codebase.

Reply:
no, sorry